docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+| IOTEDGE_IOTHUBHOSTNAME      | Host name of the IoT Hub                                                               | Yes                                 |
+| IOTEDGE_GATEWAYHOSTNAME     | Gateway host name                                                                      | Yes, if edgeHub gateway is enabled  |
+| ENABLE_GATEWAY              | Indicates whether the edgeHub gateway should be enabled or not                         | No (defaults to `true`)             |
+| IOTEDGE_DEVICEID            | The gateway deviceId                                                                   | Yes                                 |
+| HTTPS_PROXY                 | HTTPS proxy url                                                                        | No                                  |
+| RX2_DATR                    | RX2 data rate                                                                          | No (defaults to 1)                  |
+| RX2_FREQ                    | RX2 frequency                                                                          | No                                  |
+| IOTEDGE_TIMEOUT             | IoT Edge timeout in milliseconds                                                       | No (defaults to 0)                  |
+| FACADE_SERVER_URL           | Azure Facade function url                                                              | Yes                                 |
+| FACADE_AUTH_CODE            | Azure Facade function auth code                                                        | Yes                                 |
+| LOG_LEVEL                   | Logging level                                                                          | No (defaults to level 4 (Error)     |
+| LOG_TO_CONSOLE              | Indicates whether logging to console is enabled or not                                 | No (default to `true`)              |
+| LOG_TO_TCP                  | Indicates whether logging to TCP is enabled or not (used mainly for integration tests) | No (defaults to `false`)            |
+| LOG_TO_HUB                  | Indicates whether logging to IoT Hub is enabled or not                                 | No (defaults to `false`)            |
+| LOG_TO_TCP_ADDRESS          | IP address for TCP logs                                                                | Yes, only if TCP logging is enabled |
+| LOG_TO_TCP_PORT             | TCP port to send logs to                                                               | Yes, only if TCP logging is enabled |
+| NETID                       | Gateway network ID                                                                     | No (defaults to network id 1)       |
+| AllowedDevAddresses         | Allowed Dev addresses                                                                  | No                                  |
```

Comment:
Sets a list of allowed dev addresses from which message processing is enabled.

Let's say in the default that not setting it allows all messages coming from the defined NETID to be processed